bin/music_genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
```

Comment:
Please, rename the file for this class from `music_genre.rb` to `genre.rb`. Remember the file name should match the class name using the right style: `snake_case` for file and `UpperCamelCase` for class.